web/template/deputy-hub.gotmpl

```diff
@@ -0,0 +1,10 @@
+{{ template "page" . }}
+
+{{ define "title" }} OPG Sirius Deputy Hub {{ end }}
+
+{{ define "main" }}
+
+    {{/* added to pass pa11y test- will be removed in later ticket */}}
```

Comment:
If you add TODO to the comment (i.e. `{{/* added to pass pa11y test- will be removed in later ticket */}}`), it is then obvious that this needs to be done in the future.